sound/soc/sof/pcm.c

```diff
@@ -677,7 +677,8 @@ void snd_sof_new_platform_drv(struct snd_sof_dev *sdev)
 
 	pd->pcm_construct = sof_pcm_new;
 	pd->ignore_machine = drv_name;
-	pd->be_hw_params_fixup = sof_pcm_dai_link_fixup;
+	if (!sof_debug_check_flag(SOF_DBG_DSPLESS_MODE))
```

Comment:
maybe add a comment here to explain why we don't have a fixup?

Reply:
OK